README.md

```diff
@@ -53,7 +64,12 @@ local http_requests_total_counter = metrics.counter('http_requests_total')
 http_requests_total_counter:inc(1, {method = 'GET'})
 ```
 
-Using gauges:
+### Gauge
+
+Gauge is a metric that represents a single numerical value that could change arbitrarily.
```

Comment:
`could change` -> `may be changed` / `can be changed`

Reply:
done